doc/whats_new.rst

```diff
@@ -54,6 +54,8 @@ Bug fixes
 
 - ...
 
+- Fix setting unknown ages if no age parameter is supplied in csv in function :func:`mne_bids.make_report` (:gh:`912`)
```

Comment:
Should be rephrased, I don't think it's very clear what is meant here. I can give it a shot later, currently on my phone

Reply:
I tried (also to re-trigger CI), better now?

Reply:
at least CI seems to work now :)

Reply:
`Fix loading tsv participant summary information when only subject_id is supplied and all other information (age, gender, handedness) is missing in :func:`mne_bids.make_report` (:gh:`912`)`

Reply:
I pushed another small adjustment to make it more concise